protocol/protobuf/pairing.proto

```diff
@@ -255,3 +267,16 @@ message SyncContactRequestDecision {
     DECLINED = 1;
   }
 }
+
+/* `SyncProfile` is used to describe profile of logged in user */
+message SyncProfile {
+  string key_uid = 1;
+  string display_name = 2;
+  repeated SyncProfilePicture pictures = 3;
+}
+
+/* `SyncProfileSettings` is used to sync user preferences which are stored locally in the `settings` table 
+ * but for example we do not need to sync `display_name` which is one of the fields of `settings` table cause 
+ * we will already have it via `SyncProfile` */
+message SyncProfileSettings {
+}
```

Comment:
Any reason this is empty?

Reply:
answered [here](https://github.com/status-im/status-go/pull/2976#discussion_r1036277728)